requirements.txt

```diff
@@ -14,6 +14,7 @@ multidict==4.5.2
 requests==2.22.0
 six==1.12.0
 slackclient==2.1.0
+-e git+git@github.com:gmkohler/emoji-exodus.git@d78aff7bc708bc81f0be1df0e924fae4aae6b36e#egg=transfer
```

Comment:
:thinkfish:

Reply:
this is the for our new "package": the transfer script